test/unit/alignment/multiple/align_multiple_seqan2_adaptation_test.cpp

```diff
@@ -0,0 +1,90 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <string_view>
+#include <vector>
+
```

Comment:
```suggestion

```

Reply:
(We don't want redundant includes except in tutorials/snippets, so I'm removing the redundant includes I see, until this is later automated).